central/sensor/service/connection/sensorevents.go

```diff
@@ -87,10 +96,19 @@ func (s *sensorEventHandler) addMultiplexed(ctx context.Context, msg *central.Ms
 	case *central.SensorEvent_Synced:
 		// Call the reconcile functions
 		log.Info("Receiving reconciliation event")
-		if s.reconciliationMap.IsClosed() {
-			log.Infof("Ignoring SYNC from cluster %s (ClusterID:%s) since reconciliationMap is already closed", s.cluster.GetName(), s.cluster.GetId())
-			return
+
+		unchangedIDs := event.GetSynced().GetUnchangedIds()
```

Comment:
How much can this message be realistically?

Reply:
note: fallback to classic reconciliation if max message receive is received.

Reply:
Created a task to address the fallback to classic reconciliation [ROX-20878](https://issues.redhat.com/browse/ROX-20878).